sway/commands/exec_always.c

```diff
@@ -36,17 +36,12 @@ struct cmd_results *cmd_exec_process(int argc, char **argv) {
 	}
 
 	if (argc == 1 && (argv[0][0] == '\'' || argv[0][0] == '"')) {
-		tmp = strdup(argv[0]);
-		strip_quotes(tmp);
+		cmd = strdup(argv[0]);
```

Comment:
This code is missing a NULL check in case it goes out of memory.

Reply:
We don't have a OOM checking policy in Sway, yet. Many other places would need to get fixed.

Patches welcome I suppose.